platform/android/app/src/main/kotlin/coop/polypoly/polypod/MainActivity.kt

```diff
@@ -68,19 +55,49 @@ class MainActivity : AppCompatActivity(), LifecycleEventObserver {
             startActivity(Intent(this, PodUnlockActivity::class.java))
         }
 
-        if (notification.showInApp) {
+        val notificationData = UpdateNotificationData(this)
+        if (UpdateNotification.showInApp) {
             AlertDialog.Builder(this)
-                .setTitle(notification.title)
-                .setMessage(notification.text)
+                .setTitle(notificationData.title)
+                .setMessage(notificationData.text)
                 .setPositiveButton(
                     R.string.button_update_notification_close
                 ) { _, _ ->
-                    notification.handleInAppSeen()
+                    UpdateNotification.handleInAppSeen()
                 }
                 .show()
         }
     }
 
+    private fun initCore() {
+        val language = Language.determine(this@MainActivity)
```

Comment:
I just moved the code, the PR is large enough for me to not want to change anything unrelated.

Reply:
I would say it does in Android, since essentially the operating system and the browser are the same thing. Safari and iOS... probably. The point is not that they *match*, however, but to use whatever single source of truth there is for the language. Since this choice will eventually bubble up to the WebView, it would be nice to expose it that it can be picked by silly-i18n and thus all the translation strings.